pkg/annotations/annotations.go

```diff
@@ -33,6 +33,7 @@ const (
 	healthcheckTimeoutSecondsKey  = "alb.ingress.kubernetes.io/healthcheck-timeout-seconds"
 	healthyThresholdCountKey      = "alb.ingress.kubernetes.io/healthy-threshold-count"
 	unhealthyThresholdCountKey    = "alb.ingress.kubernetes.io/unhealthy-threshold-count"
+	inboundCidrsKey               = "alb.ingress.kubernetes.io/inbound-cidrs"
```

Comment:
If `inbound-cidrs` is a configuration of the security group, should this be something like `security-group-inbound-cidrs` ?